nemo_skills/pipeline/eval.py

```diff
@@ -179,7 +242,11 @@ def eval(
         help="Path to a custom dataset folder that will be searched in addition to the main one. "
         "Can also specify through NEMO_SKILLS_EXTRA_DATASETS.",
     ),
-    # extra_datasets_type: ExtraDatasetType = typer.Option(ExtraDatasetType.copy, help="How to handle extra datasets"),
+    extra_datasets_type: ExtraDatasetType = typer.Option(
+        os.getenv("NEMO_SKILLS_EXTRA_DATASETS_TYPE", "local"),
+        help="If you have extra datasets locally, set to 'local', if on cluster, set to 'cluster'."
+        "Can also specify through NEMO_SKILLS_EXTRA_DATASETS_TYPE environment variable.",
+    ),
```

Comment:
Typer has `envvar` arg in Option that reads env variables if you want that

Reply:
nice, let me use that